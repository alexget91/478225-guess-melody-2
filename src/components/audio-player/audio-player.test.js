import React from "react";
import AudioPlayer from "./audio-player";
import renderer from "react-test-renderer";
import createAudioMock from "../../common/test-stubs";

it(`Audio player correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<AudioPlayer
      src={``}
      isPlaying={false}
      onPlayButtonClick={jest.fn()}
    />, createAudioMock())
    .toJSON();

  expect(tree).toMatchSnapshot();
});
