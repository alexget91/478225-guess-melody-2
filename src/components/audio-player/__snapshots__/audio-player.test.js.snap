// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Audio player correctly renders after relaunch 1`] = `
Array [
  <button
    className="track__button track__button--play js-track-button"
    disabled={false}
    onClick={[MockFunction]}
    type="button"
  />,
  <div
    className="track__status"
  />,
]
`;
